src/app/search/armory-search.ts

```diff
@@ -63,10 +68,13 @@ export const buildArmoryIndex = memoizeOne((defs: D2ManifestDefinitions) => {
 
 export function getArmorySuggestions(
   armoryIndex: ArmoryEntry[] | undefined,
-  query: string
+  query: string,
+  language: DimLanguage
 ): ArmorySearchItem[] {
   const armoryEntries = query.length
-    ? armoryIndex?.filter((armoryItem) => armoryItem.name.toLocaleLowerCase().includes(query))
+    ? armoryIndex?.filter((armoryItem) =>
+        armoryItem.plainName.includes(plainString(query, language))
```

Comment:
Move `plainString(query, language)` out of the filter loop to avoid recomputation.